modules/ROOT/pages/resolving-conflicts.adoc

```diff
@@ -1,27 +1,26 @@
-= Resolving Conflicts
+= Conflict Resolution with 1.x client
+:page-status: deprecated
+:description: Couchbase Sync Gateway -- resolving conflicts
 :idprefix:
 :idseparator: -
 
 include::partial$_std-hdr-sgw.adoc[]
 
-Sync Gateway 2.x resolves conflicts automatically.
-This functionality aims to simplify the default behavior of conflict handling and save disk space (conflicting revisions will no longer be stored in the database).
-The Couchbase Lite SDK guides describe how the automatic conflict resolution works:
+:param-abstract: This guide relates to handling conflicts created by Couchbase Lite 1.x clients.
+include::partial$block-abstract.adoc[]
 
-
-* xref:couchbase-lite:swift:learn/swift-conflict.adoc[Swift]
-* xref:couchbase-lite:java:learn/java-conflict.adoc[Java]
-* xref:couchbase-lite:android:learn/java-android-conflict.adoc[Java (Android)]
-* xref:couchbase-lite:csharp:learn/csharp-conflict.adoc[C#]
-* xref:couchbase-lite:objc:learn/objc-conflict.adoc[Objective-C]
-
-The following guide describes how to handle conflicts that are created by Couchbase Lite 1.x clients.
-This content isn't necessary for applications that use Couchbase Mobile 2.0 only.
+== Introduction
+This content addresses conflicts created by Couchbase Lite 1.x clients.
+It describes how to handle the conflict resolution on the server-side using the Sync Gateway Admin REST API. That information does not apply to applications using Couchbase Mobile 2.x _only_ -- see {xref-sgw-pg-resolving-conflicts}.
```

Comment:
```suggestion
In these cases, the conflict resolution happens on the server-side using the Sync Gateway Admin REST API. This information does not apply to applications using Couchbase Mobile 2.x _only_ -- see {xref-sgw-pg-resolving-conflicts} or inter-Sync Gateway replication with versions of Sync Gateway 2.8 and above  -- see {xref-TBD} 
```

Reply:
I have a TBD xref that needs to point to conflict resolution in Inter Sync Gateway Replication